library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecRenderer.java

```diff
@@ -439,6 +440,10 @@ protected abstract int supportsFormat(MediaCodecSelector mediaCodecSelector,
   protected List<MediaCodecInfo> getDecoderInfos(
       MediaCodecSelector mediaCodecSelector, Format format, boolean requiresSecureDecoder)
       throws DecoderQueryException {
+    if (format.sampleMimeType == MimeTypes.VIDEO_DOLBY_VISION) {
```

Comment:
Once `format.codecs` contains the profile and `MediaCodec.getCodecProfileAndLevel` handles getting the profile for DV, this addition and the changes to `MediaCodecSelector` shouldn't be needed any more as `MediaCodecInfo.isCodecSupported` will take care of the capabilities check.

Reply:
Ok. Will upload another patch